src/libraries/System.Private.CoreLib/src/System/IO/Path.cs

```diff
@@ -75,6 +75,39 @@ public static partial class Path
                 string.Concat(subpath, ".", extension);
         }
 
+        /// <summary>
+        ///  Determines whether a given path exists on the filesystem.
+        /// true if the caller has the required permissions and the path exists on the filesystem; otherwise, false.
+        /// This method also returns false if path is null, an invalid path, or a zero-length string.
+        /// If the caller does not have sufficient permissions to read the specified path,
+        /// no exception is thrown and the method returns false regardless of the existence of path.
+        /// </summary>
+        /// <remarks>
+        /// It would also return true for existing, non-regular files like pipes unlike File.Exists.
+        /// Logically this is equivalent to File.Exists(path) || Directory.Exists(path).
+        /// </remarks>
+        /// <param name="path">The path to check</param>
+        /// <returns> Boolean </returns>
```

Comment:
```suggestion
        /// <returns>
        /// <see langword="true" /> if the caller has the required permissions and <paramref name="path" /> contains
        /// the name of an existing file or directory; otherwise, <see langword="false" />.
        /// This method also returns <see langword="false" /> if <paramref name="path" /> is <see langword="null" />,
        /// an invalid path, or a zero-length string. If the caller does not have sufficient permissions to read the specified path,
        /// no exception is thrown and the method returns <see langword="false" /> regardless of the existence of <paramref name="path" />.
        /// </returns>
```

Reply:
Fixed